src/components/router/RouterSwitch.jsx

```diff
@@ -0,0 +1,29 @@
+import { Switch, Route } from "react-router";
+import Home from "../../views/Home";
+import ProductDetail from "../../views/ProductDetail";
+import ProductsList from "../../views/ProductsList.jsx";
+import SearchResults from "../../views/SearchResults";
+
+const RouterSwitch = () => {
+    return (
+        <Switch>
+          <Route path="/search/:searchTerm">
```

Comment:
A more "elegant" way of declaring routes is like this
```suggestion
          <Route path="/search/:searchTerm" component={SearchResults} />
```

Reply:
WOW !!!
I didn't see it in docs. Commited 💯

Reply:
![image](https://user-images.githubusercontent.com/12141670/144634200-0262429d-cf89-4856-8ab0-be82ff5b563c.png)
you left 
```
 <SearchResults/>
</Route>
```
just remove it and its going to deploy correctly

Reply:
```
 <Route path="/search/:searchTerm" component={SearchResults} />
<Route path="/product/:id" component={ProductDetail} />
<Route path="/products" component={ProductsList} />
<Route path="/home" component={Home} />
<Route path="/" component={Home} />
```